crates/symbolicator/Cargo.toml

```
@@ -36,7 +36,7 @@ reqwest = { git = "https://github.com/jan-auer/reqwest", tag = "v0.11.0", featur
 rusoto_core = "0.48.0"
 rusoto_credential = "0.48.0"
 rusoto_s3 = "0.48.0"
-sentry = { version = "0.27.0", features = ["anyhow", "debug-images", "log", "tracing"] }
+sentry = { git = "https://github.com/getsentry/sentry-rust.git", branch = "master", features = ["anyhow", "debug-images", "log", "tracing"] }
```

```suggestion
sentry = { git = "https://github.com/getsentry/sentry-rust.git", branch = "master", features = ["anyhow", "debug-images", "log", "tracing", "profiling"] }
```

I think you might need to add `profiling` to _features_

---

oh jeez, you are totally right ;-)